timed/tracking/signals.py

```diff
@@ -26,11 +26,23 @@ def update_most_recent_remaining_effort(sender, instance, **kwargs):
     if kwargs.get("raw", False):  # pragma: no cover
         return
 
-    if not instance.pk:
-        return
     if instance.task.project.remaining_effort_tracking is not True:
         return
 
+	# update most_recent_remaining_effort and total_remaining_effort on report creation
+    if not instance.pk:
```

Comment:
This contains quite a bit of duplicated code. Could this be simplified?

Reply:
Created a function for the duplicated  code